client/Models/useTopics/Hook.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+import { GET_TOPICS } from 'Queries/Topics';
+import { useQuery } from '@apollo/client';
+import { useTopicsType } from './useTopics.types';
+
+export const useTopics = (): useTopicsType => {
+  const useGetTopics = () => useQuery(GET_TOPICS);
```

Comment:
I would cross reference the behviours described here: https://github.com/strimzi/strimzi-ui/issues/70#issue-733230202

I think we will need to also have a variable for topic name, which gets debounced etc

Reply:
Should also add, the consumer of the model is the view. Within reason, if there are things the model can do to make the view as simple as possible (eg if no data returning an empty array), or obfiscate (loading, error states etc) that would be good too.

Reply:
Have added client-side filter handling for the topics listing. The mockapi server doesn't support this yet, so it currently doesn't make any difference when running the webpack-dev-server, but you can see the filter input field changes are debounced correctly, by watching the network calls..